src/Type/ListType.php

```diff
@@ -37,7 +37,7 @@ public function resolve(
         foreach ($parentResult->getRawValue() as $rawValue) {
             $value = $this->innerType->createResolvedValue($rawValue);
 
-            $return[] = $value->getType()->resolve($requestedFields, $value);
+            $return[] = $this->innerType->resolve($requestedFields, $value);
```

Comment:
Task was to cover this chnage in tests.... not to revert it back with passing tests.